connections/sim/stream_logger.py

```diff
@@ -42,7 +42,7 @@ def _read_and_log(self, stream_gen):
     def _crcrlfFilter(self, stream):
         while True:
             c = stream.read(1)
-            if c == CR and stream.peek(1)[0] == LF:
+            if c[0] == CR and stream.peek(1)[0] == LF:
```

Comment:
This is actually the entire fix. The rest is just misc stuff.

Reply:
Fix aside, I'm curious how this ever worked, if at all

Reply:
Not sure, but it will never happen again (hopefully (see new commit))